app/controllers/application_controller.rb

```diff
@@ -8,4 +8,8 @@ def after_sign_in_path_for(_resource)
     dashboard_url
   end
 
+  def after_sign_out_path_for(_resource)
+    new_user_session_url
```

Comment:
If we would have one. We have now set of auth endpoints and a root path in app points to a dashboard
So we don't yet have a defined homepage, so for now it should be fine